components/ide/jetbrains/backend-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -15,11 +15,15 @@
     <!-- Product and plugin compatibility requirements -->
     <!-- https://plugins.jetbrains.com/docs/intellij/plugin-compatibility.html -->
     <depends>com.intellij.modules.platform</depends>
+    <!-- Use old formatting to resolve class not found exception -->
+    <!-- https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html#3-dependency-declaration-in-pluginxml -->
+    <depends>com.jetbrains.codeWithMe</depends>
+    <depends>org.jetbrains.plugins.terminal</depends>
     <depends optional="true" config-file="maven.xml">org.jetbrains.idea.maven</depends>
     <dependencies>
         <plugin id="Git4Idea"/>
-        <plugin id="org.jetbrains.plugins.terminal"/>
-        <plugin id="com.jetbrains.codeWithMe"/>
+        <!-- <plugin id="org.jetbrains.plugins.terminal"/> -->
```

Comment:
should not we remove it or it is required in some cases?

Reply:
I'll try if not remove it works

Reply:
I misunderstand you question. But I believe to keep new format is a good option, old format (changes we made) is about to fix SharedTerminalService class not found exception.

Not sure how we can detect such case: `this plugin requires old format` in some versions, it's a break change no 🤕?